### ISSUE TITLE
Talking to the bot (pm) kills it

### ISSUE THREAD

**Issue by tomascohen** [duck8823/Slack-RTM-Bot#36]
It seems that the way the conversations list is fetched (either the old im.list or the new conversations.list) doesn't include the bot's private conversation ID.

I haven't found on the API a way to get that.

**Comment by duck8823** [comment]
Maybe it needs an option `types` with `conversations.list`API .

https://github.com/duck8823/Slack-RTM-Bot/blob/435f3fc31c8191af8799dae6097504259cbffb91/lib/Slack/RTM/Bot/Client.pm#L52

https://api.slack.com/methods/conversations.list#arg_types

**Comment by tomascohen** [comment]
I agree! I managed to reproduce it with the old API as well (the failure).

**Comment by duck8823** [comment]
> reproduce it with the old API as well (the failure)

Hmmm...
Are bot users invited to a private channel?
